example/scripts/modifyGradle.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+// Cool script that adds props to android/build.gradle, because cordova cannot do so...
+
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function (context) {
+  const gradleFilePath = path.join(
+    context.opts.projectRoot,
+    'platforms/android/build.gradle',
+  );
+
+  if (fs.existsSync(gradleFilePath)) {
```

Comment:
Nedalo by sa tomuto zanoreniu vyhnúť takto? A podobne pre ostatné if-else
```suggestion
  if (!fs.existsSync(gradleFilePath)) {
    console.error('build.gradle file not found')
    return
  }
  
```

Reply:
toto sa vygenerovalo automaticky